src/k8s-engine/lima.ts

```diff
@@ -1392,7 +1392,7 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
                   { env: this.limaEnv, stdio: 'ignore' });
                 break;
               } catch (ex) {
-                console.log('Could not read k3s.yaml; retrying...');
+                console.log('Configuration k3s.yaml not present; will check again...');
```

Comment:
Change seems very subtle, but ok.

However, I would prefer if the log message includes the full path to the missing file:

```suggestion
                console.log('Configuration file /etc/rancher/k3s/k3s.yaml not present; will check again...');
```

Reply:
Should I specify `in lima vm`?

Reply:
> Should I specify `in lima vm`?

sure